plugins/inputs/nginx_plus/README.md

```diff
@@ -0,0 +1,125 @@
+# Telegraf Plugin: nginx_plus
+
+Nginx Plus is a commercial version of the open source web server Nginx. The use this plugin you will need a license. For more information about the differences between Nginx (F/OSS) and Nginx Plus, [click here](https://www.nginx.com/blog/whats-difference-nginx-foss-nginx-plus/).
+
+Structures for Nginx Plus have been built based on history of
+[status module documentation](http://nginx.org/en/docs/http/ngx_http_status_module.html)
+
+### Configuration:
+
+```
+# Read Nginx Plus' advanced status information
+[[inputs.nginx_plus]]
+  ## An array of Nginx status URIs to gather stats.
+  urls = ["http://localhost/status"]
+```
+
+### Measurements & Fields:
+
+- nginx_plus_processes
+  - respawned
+- nginx_plus_connections
+  - accepted
+  - dropped
+  - active
+  - idle
+- nginx_plus_ssl
+  - handshakes
+  - handshakes_failed
+  - session_reuses
+- nginx_plus_requests
+  - total
+  - current
+- nginx_plus_upstream, nginx_plus_stream_upstream
+  - keepalive
+  - zombies
+- nginx_plus_upstream_peer, nginx_plus_stream_upstream_peer
+  - requests
+  - unavail
+  - healthchecks_checks
+  - header_time
+  - response_time
+  - state
+  - active
+  - downstart
+  - healthchecks_last_passed
+  - weight
+  - responses_1xx
+  - responses_2xx
+  - responses_3xx
+  - responses_4xx
+  - responses_5xx
+  - received
+  - selected
+  - healthchecks_fails
+  - healthchecks_unhealthy
+  - backup
+  - responses_total
+  - sent
+  - fails
+  - downtime
+
+
+### Tags:
+
+- nginx_plus_processes, nginx_plus_connections, nginx_plus_ssl, nginx_plus_requests
+  - server
+  - port
+  - host
+
+- nginx_plus_upstream, nginx_plus_stream_upstream
+  - host
+  - upstream
+  - server
+  - port
+
+- nginx_plus_upstream_peer, nginx_plus_stream_upstream_peer
+  - id
+  - host
+  - upstream
+  - server
+  - port
+  - serverAddress
```

Comment:
This should be `server_address` to match the naming style.  This is the address of the `upstream`?  If so maybe it would be more descriptive to call it `upstream_address`?